src/main/java/hudson/plugins/fitnesse/FitnesseExecutor.java

```diff
@@ -68,6 +71,11 @@ public boolean execute(Launcher launcher,FilePath workspace, Run<?,?> build) thr
 				throw (InterruptedException) t;
 			return false;
 		} finally {
+			try {
+				killTest(getFitnessePage(build, false));
```

Comment:
Right, if I'm understanding the code, you're calling killTest every time but relying on a member variable to determine whether or not it should do something. Couldn't you just call killTest in the catch and not have to rely on the member variable. Feel free to correct me. I may be missing something.